packages-exp/installations-types-exp/index.d.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Provider } from '@firebase/component';
+import { _FirebaseService } from '@firebase/app-types-exp';
+
+export interface FirebaseInstallations {
```

Comment:
You should extend this interface with `_FirebaseService` and still use `FirebaseInstallations ` in ` NameServiceMapping` at the bottom of the file